### PR TITLE
chore: redirect project pausing button

### PR DESCRIPTION
**Pull request by pamelachia**
## What kind of change does this PR introduce?

Explain to free plan users that they can avoid project pausing by upgrading to pro at the project paused page. Pro plan users button remains the same. 

Screenshot 2023-08-12 at 2 53 14 PM

### CODE REVIEW

File: studio/components/layouts/ProjectLayout/ProjectPausedState.tsx [hunk 1]
```diff
@@ -128,11 +136,21 @@ const ProjectPausedState = ({ product }: ProjectPausedStateProps) => {
...
                   )}
                 </Tooltip.Root>
-                <Link href={`/project/${ref}/settings/general`}>
-                  <a>
-                    <Button type="default">View project settings</Button>
-                  </a>
-                </Link>
+                {isFreePlan ? (
+                  <Link
+                    href={`/project/${ref}/settings/billing/subscription?panel=subscriptionPlan`}
```
Comment (kevcodez) [inline studio/components/layouts/ProjectLayout/ProjectPausedState.tsx:141]: Let's have the correct URL depending on org/project level billing. We've set up some redirects, but it's still a longer route to go through the project first and then be redirected to the org. So a better UX to get the URL right

Org-Billing URL:

`/org/${slug}/billing?panel=subscriptionPlan`

To check if the org uses org level billing

```ts
 const billedViaOrg = Boolean(currentOrg?.subscription_id)
```

File: studio/components/layouts/ProjectLayout/ProjectPausedState.tsx [hunk 2]
```diff
@@ -25,6 +26,8 @@ const ProjectPausedState = ({ product }: ProjectPausedStateProps) => {
   const selectedOrganization = useSelectedOrganization()
   const { project } = useProjectContext()
   const orgSlug = selectedOrganization?.slug
+  const { data: subscription } = useProjectSubscriptionV2Query({ projectRef: ref })
+  const isFreePlan = subscription?.plan.id === 'free'
```
Comment (kevcodez) [inline studio/components/layouts/ProjectLayout/ProjectPausedState.tsx:30]:
```suggestion
  const isFreePlan = subscription?.plan?.id === 'free'
```

File: studio/components/layouts/ProjectLayout/ProjectPausedState.tsx [hunk 3]
```diff
@@ -128,11 +136,21 @@ const ProjectPausedState = ({ product }: ProjectPausedStateProps) => {
...
-                  <a>
-                    <Button type="default">View project settings</Button>
-                  </a>
-                </Link>
+                {isFreePlan ? (
+                  <Link
+                    href={`/project/${ref}/settings/billing/subscription?panel=subscriptionPlan`}
+                  >
+                    <a>
+                      <Button type="default">Change Plan</Button>
```
Comment (kevcodez) [inline studio/components/layouts/ProjectLayout/ProjectPausedState.tsx:144]: No hard feelings, just food for thought to think about different CTAs for this one

"Upgrade Plan"
"Upgrade to Pro"

Can keep it as it is though

**Review by kevcodez — APPROVED** [review]
Just a few nits, not blocking

File: studio/components/layouts/ProjectLayout/ProjectPausedState.tsx (hunk 3 above)
Comment (pamelachia) [reply studio/components/layouts/ProjectLayout/ProjectPausedState.tsx:144]: Good idea will change

**Pull request merged**